content/en/docs/concepts/workloads/pods/pod-lifecycle.md

```diff
@@ -34,10 +34,13 @@ You can also inject [custom readiness information](#pod-readiness-gate) into the
 condition data for a Pod, if that is useful to your application.
 
 Pods are only [scheduled](/docs/concepts/scheduling-eviction/) once in their lifetime.
-Once a Pod is scheduled (assigned) to a Node, the Pod runs on that Node until it stops
-or is [terminated](#pod-termination).
+You can use [Pod Scheduling Readiness](/docs/concepts/scheduling-eviction/pod-scheduling-readiness/)
+to delay scheduling for a Pod until all its _scheduling gates_ are removed. For example,
+you might want to define a set of Pods but only trigger scheduling once all the Pods
+have been created. In this example
```

Comment:
Do you mean that the phrase "In this example" forms a sentence with the following paragraph? Or does it mean something else?
<img width="614" alt="image" src="https://github.com/kubernetes/website/assets/44460091/e50655af-eb56-4e6d-83e2-5feb05236e49">

Reply:
Oops. Will fix.